go/teams/transactions.go

```diff
@@ -481,30 +481,30 @@ func resolveServerTrustAssertion(mctx libkb.MetaContext, assertion string) (upak
 	return upak, false, nil
 }
 
-func (tx *AddMemberTx) ResolveUPKV2FromAssertionOrEmail(m libkb.MetaContext, assertion string) (upak keybase1.UserPlusKeysV2, single libkb.AssertionURL, doInvite bool, err error) {
-	isServerTrustInvite, single, err := preprocessAssertion(m, assertion)
+func (tx *AddMemberTx) ResolveUPKV2FromAssertionOrEmail(m libkb.MetaContext, assertion string) (upak keybase1.UserPlusKeysV2, single libkb.AssertionURL, doInvite bool, full libkb.AssertionExpression, err error) {
```

Comment:
It's time to refactor this to return a struct instead of 5 values. Can you make a cleaning ticket?